appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -31,6 +31,7 @@
 from oauth2client.client import GoogleCredentials
 
 
+
```

Comment:
Can you remove this space? It won't pass our linter.
You can check by running:

    nox -s lint -- appengine/standard/firebase/firetactoe

from the root of the github repo. ([details](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/CONTRIBUTING.md#style))